internal/kuberesource/parts.go

```diff
@@ -552,37 +552,28 @@ func ServiceMeshProxy() *applycorev1.ContainerApplyConfiguration {
 // container to setup an encrypted LUKS mount.
 func CryptsetupInitCommand() string {
 	return `#!/bin/bash
-set -e
+set -ex
+
+# Regarding https://man7.org/linux/man-pages/man8/cryptsetup.8.html
+# passphrase <=> key in this document, not an actual cryptographic key
```

Comment:
I think the intend of the comment is not clear. Please be more descriptive in what you are trying to communicate here.